src/apps.ts

```diff
@@ -14,15 +14,15 @@ export type App = {
 
 /**
  * Get A list of all of your apps and their associated domain names
- *
  */
 export async function index(): Promise<Array<App>> {
   const apiEndpoint = setup.endpoints.api
 
   const jwt = await ucan.build({
     audience: await api.did(),
-    issuer: await did.local(),
-    proof: await localforage.getItem(UCAN_STORAGE_KEY)
+    issuer: await did.ucan(),
+    proof: undefined, // TODO
```

Comment:
still todo?

Reply:
Yeah, waiting on some API changes.
Need to add some more info here 👀

Reply:
ah cool cool :ok_hand: